rules/docker-sonic-mgmt-framework.mk

```diff
@@ -0,0 +1,25 @@
+# docker image for mgmt-framework
+
+DOCKER_MGMT-FRAMEWORK_STEM = docker-sonic-mgmt-framework
+DOCKER_MGMT-FRAMEWORK = $(DOCKER_MGMT-FRAMEWORK_STEM).gz
+DOCKER_MGMT-FRAMEWORK_DBG = $(DOCKER_MGMT-FRAMEWORK_STEM)-$(DBG_IMAGE_MARK).gz
+
+$(DOCKER_MGMT-FRAMEWORK)_PATH = $(DOCKERS_PATH)/$(DOCKER_MGMT-FRAMEWORK_STEM)
+
+$(DOCKER_MGMT-FRAMEWORK)_DEPENDS += $(REDIS_TOOLS) $(SONIC_MGMT-FRAMEWORK)
+$(DOCKER_MGMT-FRAMEWORK)_DBG_DEPENDS = $($(DOCKER_CONFIG_ENGINE_STRETCH)_DBG_DEPENDS)
+
+SONIC_DOCKER_IMAGES += $(DOCKER_MGMT-FRAMEWORK)
+$(DOCKER_MGMT-FRAMEWORK)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_STRETCH)
+$(DOCKER_MGMT-FRAMEWORK)_DBG_IMAGE_PACKAGES = $($(DOCKER_CONFIG_ENGINE_STRETCH)_DBG_IMAGE_PACKAGES)
```

Comment:
Any tools, that would be handy for debugging can be added to _DBG_IMAGE_PACKAGES.

Reply:
Did not add any tools for now. Will take feedback from the team on it. Do you have anything in mind?

Reply:
The basic tools like gdb, vm, strace. ... are already added. If you guys need any mgmt-fw specific, you could add. For example, some GO related debugging tool, if any.